api/alembic/versions/baa3e0182740_add_hfi_ready_table.py

```diff
@@ -0,0 +1,53 @@
+"""Add hfi_ready table
+
+Revision ID: baa3e0182740
+Revises: ac65354014bd
+Create Date: 2022-05-17 16:20:14.867265
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'baa3e0182740'
+down_revision = 'ac65354014bd'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic ###
+    op.create_table('hfi_ready',
+                    sa.Column('id', postgresql.UUID(as_uuid=True), nullable=False),
+                    sa.Column('hfi_request_id', sa.Integer(), nullable=False),
+                    sa.Column('planning_area_id', sa.Integer(), nullable=False),
+                    sa.Column('ready', sa.Boolean(), nullable=False),
+                    sa.Column('create_timestamp', sa.TIMESTAMP(timezone=True), nullable=False),
+                    sa.Column('create_user', sa.String(), nullable=False),
+                    sa.Column('update_timestamp', sa.TIMESTAMP(timezone=True), nullable=False),
+                    sa.Column('update_user', sa.String(), nullable=False),
+                    sa.ForeignKeyConstraint(['hfi_request_id'], ['hfi_request.id'], ),
+                    sa.ForeignKeyConstraint(['planning_area_id'], ['planning_areas.id'], ),
+                    sa.PrimaryKeyConstraint('id'),
+                    comment='Marks whether a planning area is ready for a particular HFI Request'
+                    )
+    op.create_index(op.f('ix_hfi_ready_create_timestamp'), 'hfi_ready', ['create_timestamp'], unique=False)
+    op.create_index(op.f('ix_hfi_ready_hfi_request_id'), 'hfi_ready', ['hfi_request_id'], unique=False)
+    op.create_index(op.f('ix_hfi_ready_planning_area_id'), 'hfi_ready', ['planning_area_id'], unique=False)
+    op.create_index(op.f('ix_hfi_ready_ready'), 'hfi_ready', ['ready'], unique=False)
+    op.create_index(op.f('ix_hfi_ready_update_timestamp'), 'hfi_ready', ['update_timestamp'], unique=False)
+    op.drop_index('ix_planning_weather_stations_id', table_name='planning_weather_stations')
```

Comment:
There's a problem here - why is this index being dropped?

Reply:
It's a redundant index on the primary key that was must have been manually removed from a previous PR